templates/course-theme/quiz-grade-notice.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * The Template for displaying quiz grade notice in quiz page when using Course Theme.
+ *
+ * @author      Automattic
+ * @package     Sensei
+ * @category    Templates
+ * @version     3.15.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! function_exists( 'sensei_quiz_grade_notices_map' ) ) {
+	/**
+	 * Notices map to echo notices HTML.
+	 *
+	 * @param array $notice
+	 */
+	function sensei_quiz_grade_notices_map( $notice ) {
+		// phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped -- No user data is being outputted.
```

Comment:
I think it would be nice to escape it anyway. Just to avoid issues in the future after enhancements. Maybe we could change some data on the origin without noticing that we're writing it without escaping? WDYT?

Reply:
I see what you're saying. It's always safer to escape the outputted stuff. I actually tried to setup an escape here, but could not find a simple way to do it.

The problem here is that this notice also renders a "Contact Teacher" button. It has lots of html that is stripped out by `wp_kses`. In order to preserve all of them I need to list all those html elements one by one in the options. 

I am talking about the html in here. https://github.com/Automattic/sensei/blob/master/includes/blocks/class-sensei-block-contact-teacher.php With all the elements and their attributes it becomes a long list of allowed html that needs to be passed to`wp_kses`.

Do you know any simpler way to escape these? Of do you want me to list them manually?

Also regarding this. I see that we do not escape anything that is rendered in the blocks (contact teacher button is a good example). Is that because WP is escaping them for us?

Let me know how you want me to approach this one.

Reply:
Maybe we could disable it only for the actions then. Does that make sense?

Reply:
Yup. That we can do. 9edfc953b2362eaf5a1fcc504dbfa9045bcfc7ec